docs/metrics/contradiction-score.md

```diff
@@ -0,0 +1,60 @@
+# Contradiction Score
+
+The [Cross-Encoder for Natural Language Inference](https://huggingface.co/cross-encoder/nli-deberta-v3-base) (NLI) is a
+text classification model that takes a pair of text and assigns a label: `'contradiction'`, `'entailment'` or
+`'neutral'`. Additionally, it assigns a score, expected to range around `-10` to `10` for each label. The higher the
+score, the more confident the model is to assign that label. So, it assigns the label with the highest score. This is
+useful for hallucination detection, as factual consistency implies the absence of contradictions.
+
+## Implementation Details
+
+This cross-encoder for NLI classification is based on Microsoft's
+[deberta-v3-base](https://huggingface.co/microsoft/deberta-v3-base), which was trained on NLI data, using
+[`sentence_transformers.cross_encoder`](https://www.sbert.net/docs/package_reference/cross_encoder.html).
+
+Further details for development and usage can be found on Hugging Face:
+[cross-encoder/nli-deberta-v3-base](https://huggingface.co/vectara/hallucination_evaluation_model).
+Below is a quick example of how it can be used:
+
+```py
+# Installation:
+# pip install -U sentence-transformers
+
+from sentence_transformers import CrossEncoder
+nli_model = CrossEncoder('cross-encoder/nli-deberta-v3-base')
+
+def compute_metric(ground_truth: str, inference: str) -> float:
+    scores = nli_model.predict([ground_truth, inference])
+    label = ['contradiction', 'entailment', 'neutral'][scores.argmax()]
+    return {
+        'label': label,
+        'contradiction': scores[0],
+        'entailment': scores[1],
+        'neutral': scores[2],
+    }
+
+print(compute_metric("The duck crossed the road", "The duck did not cross the road"))
+# Outputs:
+# {'label': 'contradiction', 'contradiction': 7.436936, 'entailment': -4.0519376, 'neutral': -3.030173}
+```
+
+## Examples
+| Ground Truth | Inference | Classification | Contradiction | Entailment | Neutral |
```

Comment:
One small nit:
Can we add <nobr></nobr> on Classification column?
Example:
`<nobr>Confidence ↓</nobr>`

It moves the last `n` from `contradiction` to the next line.